flake.nix

```diff
@@ -34,6 +34,7 @@
               pkgs.python310
               # used when building python distributions
               pkgs.hatch
+              (pkgs.google-cloud-sdk.withExtraComponents [ pkgs.google-cloud-sdk.components.gke-gcloud-auth-plugin ])
```

Comment:
What needs this?

Reply:
If you have credsHelpers set to gcloud for docker, pull fails. This is only an issue for us, since we have gcloud setup for our private gcr.